docs/data/sources/google-tag-manager-client-legacy.md

```diff
@@ -3,14 +3,33 @@ title: Google Tag Manager Web Template - Amplitude Analytics Legacy
 description: Collect data with ease using Amplitude Analytics Legacy GTM template - the official client-side Google Tag Manager template for seamless data collection.
 ---
 
-This is the client-side Google Tag Manager Template for Amplitude Analytics. The tag uses the [Amplitude JavaScript SDK](../sdks/javascript/index.md) for data collection.
+This is the client-side Google Tag Manager Template for Amplitude Analytics. The tag uses the [Maintenance Amplitude JavaScript SDK](../sdks/javascript/index.md) for data collection.
 
 !!!deprecated "This is a legacy template"
     This is a legacy GTM template. Use the [latest template](./google-tag-manager-client.md) for the most up-to-date implementation.
 
 !!!info Resources
     [:simple-googletagmanager: GTM Template Gallery](https://tagmanager.google.com/gallery/#/owners/amplitude/templates/amplitude-gtm-template) · [:material-github: GitHub](https://github.com/amplitude/amplitude-gtm-template)
 
+## Migration
+
+The Amplitude Analytics Legacy template, currently utilizing the [Maintenance Amplitude JavaScript SDK](../sdks/javascript/index.md) for data collection, is only slated for bug fixes until its eventual deprecation. Instead, we recommend using the [latest GTM template](../google-tag-manager-client/) that uses Amplitude's Marketing Analytics SDK for data collection. This not only ensures better support but also offers access to a host of new features.
+
+- It's important to note that the latest template is not backwards compatible with this legacy template due to the non-backwards compatibility of the internal SDK. Nonetheless, we have integrated a cookie migration logic to facilitate the transition of legacy cookies information to the latest cookie storage system.
+- For a comprehensive migration guide, please refer to the [details](../../sdks/typescript-browser/migration/).
+- To understand the behavioral differences between the old and new templates, consult the [comparison section](../../sdks/typescript-browser/migration/#comparison) in our documentation.
+
+Here are the breaking changes:
+
+| <div class="big-column">Change Area</div>  | Amplitude Analytics Browser SDK (Latest) | Amplitude Analytics Legacy |
```

Comment:
I think it would help to include header for GTM (latest) and GTM (maintenance). Maybe above the SDKs?

```suggestion
| <div class="big-column">Change Area</div>  | Amplitude GTM Client (Latest) | GTM Amplitude GTM Client (Maintenance) |
| SDK  | Amplitude Analytics Browser SDK (Latest) | Amplitude Analytics Legacy |
```

Reply:
<img width="387" alt="Screen Shot 2023-06-09 at 10 28 33 AM" src="https://github.com/Amplitude-Developer-Docs/amplitude-dev-center/assets/13028329/e475d5db-9e9c-4282-b272-8fdad7a788af">

Reply:
How about this?
<img width="802" alt="Screen Shot 2023-06-09 at 10 43 09 AM" src="https://github.com/Amplitude-Developer-Docs/amplitude-dev-center/assets/13028329/64759354-8bb9-4612-afd0-c021a19cbb2b">